Samples/excel-add-in-chart-analyze-data/src/taskpane/taskpane.ts

```diff
@@ -0,0 +1,588 @@
+  } catch (error) {
+    showStatus(error, true);
+  }
+}
+
+//TODO:reverse plot order
+async function reversePlotOrder() {
+  try {
+    await Excel.run(async (context) => {
+      showStatus("Running", false);
+      const sheet = context.workbook.worksheets.getItem("Sample");
+
+      let chart = sheet.charts.getItem("salesLocation");
+
+      chart.axes.categoryAxis.reversePlotOrder = true;
+      chart.axes.valueAxis.displayUnit = "Thousands";
+
+      await context.sync();
+      showStatus('Success for "Reverse vertical axis order"', false);
+    });
+  } catch (error) {
+    showStatus(error, true);
+  }
+}
+
+async function moveChartTitle() {
+  try {
+    await Excel.run(async (context) => {
+      showStatus("Running", false);
+      const sheet = context.workbook.worksheets.getItem("Sample");
+
+      let chart = sheet.charts.getItem("salesLocation");
+      chart.legend.position = "Bottom";
+      chart.legend.left = 0;
+      chart.legend.top = 230;
+      chart.legend.width = 80;
+      chart.legend.height = 20;
+      chart.title.left = 8;
+      chart.title.top = 8;
+      chart.title.setFormula("=Sample!L37");
+      await context.sync();
+      showStatus('Success for "Customize title and legend"', false);
+    });
+  } catch (error) {
+    showStatus(error, true);
+  }
+}
+
+async function showCorrelation1() {
+  try {
+    await Excel.run(async (context) => {
+      showStatus("Running", false);
+      const sheet = context.workbook.worksheets.getItem("Sample");
+
+      let dataRange = sheet.getRange("A40:A41");
+      let chart = sheet.charts.add("XYScatter", dataRange, "auto");
+      chart.name = "correlation1";
+      chart.title.text = "Correlation betweeen Sales and Temperature";
+      chart.title.format.font.size = 12;
+      chart.title.left = 8;
+      chart.title.top = 8;
+      chart.left = 600;
+      chart.top = 250;
+      chart.width = 300;
+      chart.legend.visible = false;
+      chart.setPosition("K16", "P30");
+
+      let seriesCollection = chart.series;
+      let series = seriesCollection.add("Sales and Temperature");
+
+      let rangeX = "E2:E33";
+      let xValue = sheet.getRange(rangeX);
+      series.setXAxisValues(xValue);
+
+      let rangeY = "H2:H33";
+      let value = sheet.getRange(rangeY);
+      series.setValues(value);
+
+      seriesCollection.getItemAt(0).delete();
+
+      chart.axes.valueAxis.displayUnit = "Thousands";
+
+      //Show trendline
+      series.trendlines.add("Exponential");
+      let trendline = series.trendlines.getItem(0);
+      trendline.displayRSquared = true;
+
+      //High light title
+      var font = chart.title.getSubstring(21, 5).font;
+      var font2 = chart.title.getSubstring(31, 11).font;
+      font.color = "#FF7F50";
+      font2.color = "#FF7F50";
+      await context.sync();
+      showStatus('Success for "Sales/Temper correlation"', false);
+    });
+  } catch (error) {
+    showStatus(error, true);
+  }
+}
+
+async function hightlight1() {
```

Comment:
Typo in the function name. Also, consider clarifying what's being highlighted, instead of numbering these methods so the code is easier to follow.
```suggestion
async function highlight1() {
```